test/helpers/add_retries_test_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Usage: Extend a descendant of Minitest::Test with this module
+# and then add "__Nretries" to the end of the names of the tests you want to retry up to N times if they fail
+module AddRetriesTestHelper
+  def run_one_method(klass, method_name, reporter)
+    match_data = method_name.match(/__(?<retries>\d+)retries$/)
+    return super unless match_data && match_data[:retries]
+    retries = match_data[:retries].to_i
+
+    (retries + 1).times do |i|
+      print(" Retry#{i}/#{retries}:") if i > 0 && !ENV["VERBOSE"]
```

Comment:
This is how retries look locally:

<img src="https://user-images.githubusercontent.com/4789493/53674282-d9c80b80-3c5a-11e9-8cac-79167fc6272c.png" width="200px" />